test/trace_files/psp/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Note: list of traces is created at cmake time, not build time
```

Comment:
Missing license header.

Please also format it according to the `.cmake-format` file in the project root

Reply:
We don't have licensing for any of the CMakeLists.txt files below test, and the licensing for CMakeLists.txt in other parts of the repo isn't complete either. Are those files ones that need licensing?

Reply:
I did a global run through cmake-format, although it changed a lot of other CMakeLists.txt files that I didn't change.

Reply:
Mark I was meaning to format only this new CmakeFile! :)
Also because running a global run of cmake-format and clang-format modified 111 files (a lot of files that you didn't change). This makes difficult to read the PR changes for the visitors (when we'll link this PR into the changelog for example).